projects/packages/jetpack-mu-wpcom/src/features/replace-site-visibility/replace-site-visibility.php

```diff
@@ -11,8 +11,8 @@
  * @return bool
  */
 function is_jetpack_connected() {
-	// @phan-suppress-next-line PhanUndeclaredClassMethod, PhanUndeclaredClassInCallable
-	return class_exists( 'Jetpack' ) && is_callable( 'Jetpack::is_connection_ready' ) && Jetpack::is_connection_ready();
+	// @phan-suppress-next-line PhanUndeclaredClassMethod
```

Comment:
Looks like a reasonable change, but doesn't seem to have anything to do with the PHP 7.2 change.

In this case, I don't see any indication as to why the `is_callable` was added in a4dea908f50d6e5cb964abd4f3f5650b07c92d7f, although Phan may again be the reason.

The tricky part with adding a stub is that Simple has its own Jetpack class (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Spynff.wrgcnpx.cuc%3Se%3Q37575q28-og) which doesn't entirely match Jetpack's, so whether we point to Jetpack's or make a stub from Simple's it'd be wrong for some environments. :sigh:

Reply:
Updated the PR title. Good to know about the Simple stub! At least in this case it still should be safe.